_posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md

```diff
@@ -13,4 +13,123 @@ tags:
 teaser:
     info: An easy way to create a mobile app from an existing Laravel application
 ---
-This tutorial describes an easy way to create a mobile app from your existing Laravel application.
+This tutorial describes an easy way to create a mobile app from your existing Laravel application. At the end of this tutorial, you will be able to create a hybrid app from an
```

Comment:
Maybe add a quick Intro on why using Laravel, versus other tools ? Added benefits or link to a page listing those ?